src/dodal/beamlines/i22.py

```diff
@@ -77,16 +138,17 @@ def synchrotron(
     )
 
 
+# d12 at p38
 def waxs(
```

Comment:
I'd rather see this layed out like saxs, it's clearer what's going on

Reply:
it is not obvious that it would be clearer